docs/visual-basic/developing-apps/programming/computer-resources/how-to-start-an-application-and-send-it-keystrokes.md

```diff
@@ -9,17 +9,21 @@ helpviewer_keywords:
 ms.assetid: f1303184-fce4-44fb-88b4-aac5f42d5d77
 ---
 # How to: Start an Application and Send it Keystrokes (Visual Basic)
-This example uses the `Shell` function to start the calculator application and then multiplies two numbers by sending keystrokes using the `My.Computer.Keyboard.SendKeys` method.  
-  
-## Example  
- [!code-vb[VbVbalrMyComputer#25](~/samples/snippets/visualbasic/VS_Snippets_VBCSharp/VbVbalrMyComputer/VB/Class2.vb#25)]  
-  
-## Robust Programming  
- A <xref:System.ArgumentException> exception is raised if an application with the requested process identifier cannot be found.  
-  
-## .NET Framework Security  
- The call to the `Shell` function requires full trust (<xref:System.Security.SecurityException> class).  
+
+This example uses the `Shell` function to start the Notepad application and then prints a sentence by sending keystrokes using the `My.Computer.Keyboard.SendKeys` method.
```

Comment:
I think using `xref`s here would be better. I would also like to either use the word "function" or the word "method".